test/parse.js

```diff
@@ -19,7 +19,12 @@ var Path = require('path');
 // http://mxr.mozilla.org/mozilla-central/source/js/src/tests/ecma_5/JSON/
 
 var dirsPath = Path.resolve(__dirname, 'parse-cases');
-var dirs = FS.readdirSync(dirsPath);
+var dirs = FS.readdirSync(dirsPath).filter(function(fn) {
+  if (fn.startsWith('.')) {
```

Comment:
I made the change in my tree FWIW:
https://github.com/ianloic/json5/commit/65fc0bbe2118a5da192ad7b34238d07227677b39